app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,39 @@
+class MenusController < ApplicationController
+  before_action :build_menu!, only: %i[new create]
+  before_action :find_menu!, only: %i[edit update destroy]
+
+  def index
+    @menus = Menu.all
+  end
+
+  def show
+    menu   = Menu::Show.call(params)
+    @menu  = menu[:menu].decorate
+    @order = menu[:order]
```

Comment:
Controller must return one Presenter(Facade) that includes all data that you need on view from server.
[source](https://medium.com/@ryakh/facade-pattern-on-rails-d65b86cdb5b1)

Reply:
Call `menu.order` instead `menu[:order]`
Decorator just extend model, don't cut back the functionality